code/game/objects/items/implants/implant.dm

```diff
@@ -17,6 +17,9 @@
 /obj/item/implant/proc/on_death(emote, mob/living/carbon/source)
 	return
 
+/obj/item/implant/proc/on_life(mob/living/carbon/source)
+	return

```

Comment:
It'd be more performant to just make it use `process()` imo. @vuonojenmustaturska thought?